### ISSUE TITLE
Live deployment is dead

### ISSUE THREAD

**Issue by Yokutto** [TrilonIO/zeit-now-nestjs#7]
The title says it all...

The URL https://nestjs-zeit.now.sh is showing `DEPLOYMENT_NOT_FOUND` error :(

**Comment by Yokutto** [comment]
I think it is caused by: https://github.com/TrilonIO/zeit-now-nestjs/commit/d187085b26e49f921d2886a9a63d10ffbb2ab019#commitcomment-38228725

**Comment by ScreamZ** [comment]
Looks like a vercel breaking change, I just reported the issue https://github.com/zeit/now/discussions/4333

@kamilmysliwiec maybe you have some ideas, I'll keep in touch when I get more information

Also files needs (it's currently backward compatible but for how long?) to be renamed `vercel.json` et builder `@vercel/node`

**Comment by ScreamZ** [comment]
@Yokutto I managed to make this repository works with 19.0.1 but I cannot make my own app work…

**Comment by Yokutto** [comment]
@ScreamZ I followed the discussion in vercel repository. If I understand correctly, vercel does not support asynchronous functions anymore, so I can't wait for nestjs to start, is there any way to get support for nestjs again at vercel in the future? 🤔

**Comment by ScreamZ** [comment]
I don't think so, it will work as long as your app boot fast,but then it's totally random race condition, you should use app engine or heroku for nestjs without config, or try the azure function driver

**Comment by jlarmstrongiv** [comment]
It’s possible, just not pretty 😂 working on a promising proof of concept https://www.npmjs.com/package/create-vercel-http-server-handler and example project https://github.com/jlarmstrongiv/next-with-nest-graphql
